sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2386,7 +2404,7 @@ export interface PageBlobUploadPagesFromURLOptions extends CommonOptions {
     conditions?: PageBlobRequestConditions;
     customerProvidedKey?: CpkInfo;
     encryptionScope?: string;
-    sourceConditions?: ModifiedAccessConditions;
+    sourceConditions?: MatchConditions & ModificationConditions;
```

Comment:
>MatchConditions & ModificationConditions; [](start = 23, length = 41)

You are changing `ModifiedAccessConditions` to `MatchConditions & ModificationConditions` because UploadPages doesn't support tags conditional header?

Reply:
x-ms-source-if-tags condition to check source tags 
- Works only on Copy Blob (Async) 
- Does not work with AppendBlobfromURI, PutPagefromURI, CopyBlobfromURL 

Though they may change this serveral months later.